accelerator/README.rst

```diff
@@ -1,7 +1,7 @@
 Accelerator
 ============
 
-The accelerator role allows users to  set up the `AMD ROCm <https://www.amd.com/en/graphics/servers-solutions-rocm>`_ platform or the `CUDA Nvidia toolkit <https://developer.nvidia.com/cuda-zone>`_. These tools allow users to unlock the potential of installed GPUs.
+The accelerator role allows users to set up the `AMD ROCm <https://www.amd.com/en/graphics/servers-solutions-rocm>`_ platform, the `CUDA Nvidia toolkit <https://developer.nvidia.com/cuda-zone>`_ or the `Intel Gaudi <https://docs.habana.ai/en/latest/index.html>`_ platform. These tools allow users to unlock the potential of installed GPUs.
```

Comment:
doc will be updated by omnia doc team, request to remove this

Reply:
Shouldn't we keep atomicity on this? Why don't have all the core modifications regarding Gaudi3 SW support in one place? In this way we avoid issues related to features that are not fully complete. We don't want to miss the documentation. Can I suggest pushing this as part of this PR and then if there is a need for other changes you can do them in a new PR?

Reply:
Agreed to revert this one